src/containers/study/ParticipantsTable.js

```diff
@@ -66,6 +67,10 @@ const ParticipantsTable = (props :Props) => {
   const deleteParticipantRS :?RequestState = useRequestState(['studies', DELETE_STUDY_PARTICIPANT]);
   const changeEnrollmentStatusRS :?RequestState = useRequestState(['studies', CHANGE_ENROLLMENT_STATUS]);
 
+  useEffect(() => {
+    document.getElementsByTagName('table')[0].style.minWidth = '960px';
+  }, []);
```

Comment:
This looks super hacky and also likely to break the page since you're not doing `?.`. What are you trying to do? And why can't you do this with a styled component? For example:

```
const TableWrapper = styled.div`
  table {
    minWidth: 960px;
  }
`
```

Reply:
Switched to styled